src/renderer/fileListView/fileListHeader.tsx

```diff
@@ -85,12 +90,14 @@ export const FileListHeader = (props: IFileListHeaderProps) => {
         return (
             <div className='fileView_Header'>
                 <div className='fileView_Header_Text'>
-					Optimization complete! You've saved{' '}{calculateToMb(siteImageData.compressedImagesOriginalSize-siteImageData.compressedTotalSize)}{' '}MB of space.
+					Optimization complete! You've saved{' '}{convertBytesToMb(
+						compressedImagesOriginalSize - compressedTotalSize
+					)}{' '}MB of space.
```

Comment:
This value is now calculating compression savings of all time, instead of just the current optimization session. We'll want to probably add a new state item for this.

Reply:
Dang, good catch. Fixing that up!